Pear/Cells/CommunityUserTableViewCell.swift

```diff
@@ -105,10 +105,12 @@ class CommunityUserTableViewCell: UITableViewCell {
                 interests = userInterests
                 interestsCollectionView.reloadData()
             }
-            if let profilePictureURL = user.profilePictureURL,
-               let userGoogleId = user.googleID {
-                profileImageView.kf.setImage(with: Base64ImageDataProvider(base64String:  profilePictureURL, cacheKey: userGoogleId))
-            }
+            /*
```

Comment:
I'm commenting this out for now because our server has some outdated users with the wrong profile photo string, which causes app to crash. Will comment back in when backend does this.